conf/references/production_reg_conf.pl

```diff
@@ -49,6 +49,22 @@
 # before loading all Non-Vertebrates
 Bio::EnsEMBL::Registry->load_registry_from_url("mysql://ensro\@mysql-ens-mirror-3:4275/$curr_release");
 
+# Used for protist collection databases
+Bio::EnsEMBL::Compara::Utils::Registry::load_collection_core_database(
+    -host   => 'mysql-ens-sta-3',
+    -port   => 4160,
+    -user   => 'ensro',
+    -pass   => '',
+    -dbname => "protists_choanoflagellida1_collection_core_${curr_eg_release}_${curr_release}_1",
```

Comment:
`${curr_eg_release}` needs to be defined, preferably around line 35

And these collection dbs need to be included in the section for `PREVIOUS CORE DATABASES` (below line 68) with `${prev_eg_release}_${prev_release}` - so `${prev_eg_release}` will also need defining

Reply:
:facepalm: My fault, I forgot completely about that part when helping Lahcen to define these :sweat_smile:

Reply:
as for `${curr_eg_release}`, if we're talking about the released data, `curr_eg_release = curr_release - 53`. should work